Aquality.Selenium.Core/src/Aquality.Selenium.Core/Configurations/ILoggerConfiguration.cs

```diff
@@ -10,5 +10,25 @@ public interface ILoggerConfiguration
         /// </summary>
         /// <value>Supported language.</value>
         string Language { get; }
+
+        /// <summary>
+        /// Perform page source logging in case of catastrophic failures or not.
+        /// </summary>
+        bool LogPageSource { get; }
+
+        /// <summary>
+        /// Do log attribute value or not.
+        /// </summary>
+        bool LogAttributeValue { get; }
+
+        /// <summary>
+        /// Do log text value or not.
+        /// </summary>
+        bool LogTextValue { get; }
+
+        /// <summary>
+        /// Do log wait for element's state or not.
+        /// </summary>
+        bool LogWaitForState { get; }
```

Comment:
Why do we need these values in configuration? I beleive most end-users won't use it. We can just log info and that is it.

Reply:
removed